app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb

```diff
@@ -24,16 +26,11 @@ def each_batch
   private
 
   def filter_fields
```

Comment:
@douglasgabriel You're right, I changed it so these values can be inside of a constant. I also changed the `filter_fields` assignment to avoid repetiton.